server/rematch/settings.py

```diff
@@ -23,19 +24,24 @@
 SECRET_KEY = 'qvc7t@rd5#1l-n_%%&+_fu+-lu#sp2oonf9mto%bn-1#i7$(tu'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+# FIXME
```

Comment:
Celeryd has a leak in debug mode which drinks up all memory.
The trick to stay in debug mode for the server and still run celery w/o debugging is to check
whether or not "celery" or "celeryd" is in sys.argv, if it is - set DEBUG variable to False.